test/integration/index.test.js

```diff
@@ -79,6 +86,7 @@ describe('Pullie (integration)', function () {
       assume(res).hasOwn('statusCode', 200);
       assume(body).hasOwn('status', 'ok');
       assume(nock.isDone()).is.true();
+      assume(requestsProcessed.find(req => req.repository === 'org/repo' && req.number === 165)).is.truthy();
```

Comment:
What is this `165`?

Reply:
It's from [this fixture](https://github.com/godaddy/pullie/blob/master/test/fixtures/payloads/open-pr.json#L3)